hack/make-rules/helm.mk

```diff
@@ -29,24 +32,21 @@ helm-install: $(TOOLBIN)/helm
 
 .PHONY: helm-chart-push
 helm-chart-push: helm-login $(TOOLBIN)/helm
-	$(ABSTOOLBIN)/helm chart save ../${CHART} ${IMG}
-	$(ABSTOOLBIN)/helm chart list ../${CHART}
-	$(ABSTOOLBIN)/helm chart push ${IMG}
-	$(ABSTOOLBIN)/helm chart remove ${IMG}
+	$(ABSTOOLBIN)/helm package ../${CHART} --version=${HELM_TAGNAME} --destination=${TEMP}
```

Comment:
should we add --app-version here ?

Reply:
I think that version is enough but if you think otherwise I can add it. The version should be the same as the chart tag so if we would like to push the chart with different tags we can change the version accordingly. 
A side note - this makefile is currently used only in testing and in particular for the fybrik-template chart under the modules directory.  it was used to push the implicit copy module charts but they were recently moved to https://github.com/fybrik/data-movement-operator/tree/master/modules from the fybrik repo.

Reply:
I think not but I can add it if you think otherwise. Version flag was added to enable pushing the chart with different tags as the version number must be the same as the chart tag.

Reply:
--app-version is different from --version. 
The first one is an application version in the chart [chart-versions-and-appversions](https://codefresh.io/docs/docs/new-helm/helm-best-practices/#chart-versions-and-appversions).
Maybe in our case, we should set both flags to ${HELM_TAGNAME}.
What do you think ?

Reply:
the --app-version flag (the version of the application in the chart) is different from --version`(the chart version). But it is not related to the PR, and can be a separate issue, so let't leave for now.

Reply:
A question: why do we need --version and --app-version in the Makefile? Both are defined for a relevant chart, for example: https://github.com/fybrik/fybrik/blob/cba2b8e35ee45b3668131b224b801407ce3f1ab3/modules/fybrik-template/Chart.yaml#L26 and https://github.com/fybrik/fybrik/blob/cba2b8e35ee45b3668131b224b801407ce3f1ab3/modules/fybrik-template/Chart.yaml#L21

Reply:
Hi @shlomitk1, thanks for pointing on the Chart.yaml. 
Please correct me, but I think it's better to have the `--version` flag in the Makefile ( it overwrites the settings in the Chart.yam file).
The chart version is a part of the packaged chart file name. We need it in the next command in `helm.mk`
However, I'd like to ask another question, do we really need to specify `appVersion`, both in the `Chart.yaml` and `helm.mk` files. What does the number 1.16.0 mean? 
One of the recommendations from helm [best practices](https://codefresh.io/docs/docs/new-helm/helm-best-practices/#simple-1-1-versioning) is to use the same version for chart and app.   
Maybe we can use the fybrik release version as the chart and app version, now it's 0.5.
Should it be a separate discussion ?

Reply:
@roytman @shlomitk1 fyi - the charts in fybrik are not pushed using `helm.mk`but rather published to the [`charts`](https://github.com/fybrik/charts) repo (https://github.com/fybrik/fybrik/blob/master/.github/workflows/chart-publish.yaml). `helm.mk` is currently used in testing and in particular to push `fybrik-template` module chart to the kind cluster in testing. it was used to push the implicit-copy modules before they were moved to https://github.com/fybrik/data-movement-operator/tree/master/charts

Reply:
@roytman regarding fybrik charts: after a release branch is created the appVersion and Version is updated. please see https://github.com/fybrik/fybrik/blob/master/.github/workflows/release-branch.yaml#L17

Reply:
@roytman I will add --app-version as well to `helm.mk`. Thanks

Reply:
@revit13 @roytman I was not referring to the fybrik charts as they are not pushed using the Makefile command.
I was thinking of various modules that can be added to fybrik repo (currently, it's only fybrik-template but could be more in the future). If I remember correctly, the version for all modules is synchronized with the Fybrik release whereas the appVersions can be different.

Reply:
As suggested by @roytman  I removed appVersion from Chart.yaml of fybrik-template. which in this case helm sets it to the same value as version.